src/test/java/step2/Step2.java

```diff
@@ -0,0 +1,71 @@
+package step2;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class Step2 {
+
+
+    private Calculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new Calculator();
+    }
+
+    @DisplayName("더하기 테스트")
+    @Test
+    public void 더하기테스트(){
```

Comment:
이미 `@DisplayName` 에서 해당 메서드에 대해 테스트 내용을 잘 설명해주시고 계시기 때문에
메서드 네이밍을 간략하게 변경해도 좋을 것 같아요 😄 
```suggestion
    @DisplayName("더하기 테스트")
    @Test
    public void plus() {
```

Reply:
아하 명시하면 메서드명은 중복되는것처럼 안해도 되는군요 !